udaykalra/portfolio/src/main/webapp/script.js

```diff
@@ -12,17 +12,24 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+/** Creates an <li> element containing text. */
```

Comment:
Changing:

"containing text" -> "containing the provided `inputText`."

would be more specific.